web_src/js/features/repo-common.js

```diff
@@ -3,31 +3,33 @@ import {hideElem, showElem} from '../utils/dom.js';
 import {POST} from '../modules/fetch.js';
 
 async function getArchive($target, url, first) {
+  const dropdownBtn = $target[0].closest('.ui.dropdown.button');
+
   try {
+    dropdownBtn.classList.add('is-loading');
     const response = await POST(url);
     if (response.status === 200) {
       const data = await response.json();
       if (!data) {
         // XXX Shouldn't happen?
-        $target.closest('.dropdown').children('i').removeClass('loading');
+        dropdownBtn.classList.remove('is-loading');
```

Comment:
The `loading` class is still referenced here:
```css
templates/repo/graph.tmpl:53:                   <div class="ui segment loading gt-hidden" id="loading-indicator"></div>
web_src/css/features/gitgraph.css:7:#git-graph-container > .ui.segment.loading {
web_src/css/base.css:774:.ui.loading.segment::before,
web_src/css/base.css:775:.ui.loading.form::before {
web_src/css/base.css:779:.ui.loading.form > *,
web_src/css/base.css:780:.ui.loading.segment > * {
web_src/css/base.css:875:.ui.loading.loading.input > i.icon svg {
web_src/js/features/comp/WebHookEditor.js:38:    this.classList.add('loading', 'disabled');
}
```

Reply:
In other words: You can remove even more dead code :grin:

Reply:
Good find, I only search the JS this time.

Reply:
All done.